.github/workflows/zenhub-close.yml

```diff
@@ -0,0 +1,24 @@
+name: Zenhub Close Issue Ready for Prod
+
+on:
+  push:
+    branches:
+      - prod
+
+jobs:
+  # run the python script to close the issue in zenhub
+  close-issue:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+      - name: Close Issue
+        run: |
+          pip install requests
+          python3 ./scripts/zenhub_close.py >> $GITHUB_STEP_SUMMARY
+        env:
+          ZENHUB_TOKEN: ${{ secrets.ZENHUB_TOKEN }}
```

Comment:
We can accept this for now, but this is actually a prime example of why it'd be good to move to a single GHA for our entire workflow: we would want to gate the execution of this job on successful passing of all required tests/checks that run post-deploy.